pom.xml

```diff
@@ -411,6 +411,20 @@ under the License.
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.cyclonedx</groupId>
+                <artifactId>cyclonedx-maven-plugin</artifactId>
+                <version>2.7.3</version>
```

Comment:
Can the version and plugin configuration be managed by the parent instead? This (for now) is in @zentol 's github https://github.com/zentol/flink-connector-parent/blob/master/pom.xml

Reply:
Hey @dannycranmer! Thank you for your comment. Yeah, in the meantime, we had some conversation with @zentol about that. There are 2 ways to do it: (1) I will wait until he will do his commits into the `flink-connector-shared-utils` repo, or (2) I can directly open a PR into his repo, what you suggested as well. But yeah, all-in-all I will close this PR anyway